pkg/ui/workspaces/cluster-ui/src/components/liveDataPercent/liveDataPercent.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+import { Row } from "antd";
+import React from "react";
+
+import { Bytes, Percentage } from "src/util";
+
+type Props = {
+  // Float between 0-1.
+  liveBytes: number;
+  totalBytes: number;
+};
+
+export const LiveDataPercent: React.FC<Props> = ({ liveBytes, totalBytes }) => {
+  return (
+    <div>
+      <Row justify={"end"}>
+        {Percentage(liveBytes, liveBytes ? totalBytes : 1, 1)}
```

Comment:
nit: Am I correct in saying that this is trying to handle the case where if liveBytes == 0, Percentage will return `0%` instead of `--%`? if so, maybe this will make that more clear:

```suggestion
        {liveBytes ? Percentage(liveBytes, totalBytes, 1) : '0%'}
```